components/callbacks/statemachine_test.go

```diff
@@ -91,7 +91,7 @@ func TestValidTransitions(t *testing.T) {
 	cbTask := out.Tasks[0].(callbacks.InvocationTask)
 	require.Equal(t, "http://address:666", cbTask.Destination)
 
-	// Store the pre-succeeded state to test Failed later
+	// Store the pre-succeeded state to test failed later
```

Comment:
No need to change this, `Failed` is part of the transition name.

Reply:
Whoops. Probably some bad IDE auto-refactor. Reverted.